tests/MySqlTest.php

```diff
@@ -110,7 +122,7 @@ public function it_can_generate_a_dump_command_for_specific_tables_as_array()
             ->includeTables(['tb1', 'tb2', 'tb3'])
             ->getDumpCommand('dump.sql', 'credentials.txt');
 
-        $this->assertSame('"mysqldump" --defaults-extra-file="credentials.txt" --skip-comments --extended-insert dbname tb1 tb2 tb3 > "dump.sql"', $dumpCommand);
+        $this->assertSame('"mysqldump" --defaults-extra-file="credentials.txt" --extended-insert --skip-comments dbname tb1 tb2 tb3 > "dump.sql"', $dumpCommand);
```

Comment:
Could you modify the code that it does provide exactly the same output as before? `--skip-comments` should come before `--extended-insert`.